index.html

```diff
@@ -1,106 +1,104 @@
+      <div class="feature features__item">
+        <img src="assets/images/feature-free.svg" alt="" class="feature__icon">
+        <p class="feature__text">Darmowa</p>
       </div>
+      <div class="feature features__item">
+        <img src="assets/images/feature-easy.svg" alt="" class="feature__icon">
+        <p class="feature__text">Łatwa w osłudze</p>
       </div>
-    </section>
-    <section class="alinka-section download">
-      <h3 class="titles download__title">Zacznij pracować z Alinką!</h3>
-      <div class="download__buttons">
-        <button class="button button--vivid">Pobierz</button>
+      <div class="feature features__item">
+        <img src="assets/images/feature-safe.svg" alt="" class="feature__icon">
+        <p class="feature__text">Bezpieczna</p>
       </div>
-      <img src="assets/images/download-graphics.svg" alt="" class="download__graphics">
-    </section>
-    <footer class="footer">
-      <img src="assets/images/Code_for_Poznan_final-01.png" alt="Code for Poznan logo" class="footer__logo" />
-      <div class="icons footer__contact">
-        <p class="icons__email">hello@codeforpoznan.pl</p>
-        <div class="social-media icons__social-media">
-          <a href="https://www.facebook.com/CodeForPL/" class="social-media__item">
-            <img src="assets/images/facebook.svg" alt="Facebook" />
-          </a>
-          <a href="https://pl.linkedin.com/company/codeforpoznan" class="social-media__item">
-            <img src="assets/images/linked-in.svg" alt="linked-in" />
-          </a>
-          <a href="https://github.com/CodeForPoznan" class="social-media__item">
-            <img src="assets/images/github.svg" alt="Github-link" />
-          </a>
-        </div>
+    </div>
+    </div>
+  </section>
+  <section class="alinka-section download">
+    <h3 class="titles download__title">Zacznij pracować z Alinką!</h3>
+    <div class="download__buttons">
+      <button class="button button--vivid">Pobierz</button>
+    </div>
+    <img src="assets/images/download-graphics.svg" alt="" class="download__graphics">
+  </section>
+  <footer class="footer">
+
+    <img src="assets/images/Code_for_Poznan_final-01.png" alt="Code for Poznan logo" class="footer__logo" />
+    <div class="icons footer__contact">
+      <p class="icons__email">hello@codeforpoznan.pl</p>
+      <div class="social-media icons__social-media">
+        <a href="https://www.facebook.com/CodeForPL/" class="social-media__item">
+          <img src="assets/images/facebook.svg" alt="Facebook" />
+        </a>
+        <a href="https://pl.linkedin.com/company/codeforpoznan" class="social-media__item">
+          <img src="assets/images/linked-in.svg" alt="linked-in" />
+        </a>
+        <a href="https://github.com/CodeForPoznan" class="social-media__item">
+          <img src="assets/images/github.svg" alt="Github-link" />
+        </a>
       </div>
-    </footer>
-  </body>
+    </div>
+
+  </footer>
+</body>
+
```

Comment:
remove unneeded spaces between tags in a whole file

Reply:
done